float64 Speed
float64 Steering
float64 Braking
uint8 Gear
bool Handbrake
bool RightLight
bool LeftLight
bool FrontLight
bool DriveMode
uint8 SpeedMode
uint8 SteerMode
bool Advanced
bool Emergency

uint8 Gear_Forward = 1
uint8 Gear_Neutral = 2
uint8 Gear_Reverse = 3

bool Handbrake_Off = 0
bool Handbrake_On = 1

bool Light_Off = 0
bool Light_On = 1

bool DriveMode_Remote = 0
bool DriveMode_Autonomous = 1

uint8 SpeedMode_Fast = 0
uint8 SpeedMode_Slow = 1

uint8 SteerMode_Opposite = 1
uint8 SteerMode_Normal = 2
uint8 SteerMode_Synchro = 3

bool Advanced_Off = 0
bool Advanced_On = 1

bool Emergency_Off = 0
bool Emergency_On = 1



